src/main/java/eu/accesa/teamview/service/impl/DefaultTeamMemberService.java

```diff
@@ -1,34 +1,58 @@
 package eu.accesa.teamview.service.impl;
 
+import eu.accesa.teamview.client.TeamClient;
+import eu.accesa.teamview.model.Team;
 import eu.accesa.teamview.model.TeamMember;
 import eu.accesa.teamview.repository.TeamMemberRepository;
+import eu.accesa.teamview.repository.TeamRepository;
 import eu.accesa.teamview.service.TeamMemberService;
 import lombok.NonNull;
 import org.springframework.stereotype.Service;
 
 import javax.persistence.EntityNotFoundException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 
-@Service
+
 public class DefaultTeamMemberService implements TeamMemberService {
```

Comment:
what's the purpose of this DefaultTeamMemberService? And why is no longer a @Service ?